demo/malloy-demo-composer/scripts/package.ts

```diff
@@ -12,47 +12,79 @@
  */
 
 /* eslint-disable no-console */
-
+import fs from "fs";
 import packager from "electron-packager";
 import { exit } from "process";
 import { doBuild } from "./build";
+import * as path from "path";
+import { exec } from "child_process";
+
+const outDir = "./dist";
+const thirdPartyNotices = "third_party_notices.txt";
 
-async function packageDemo(platform: string, architecture: string) {
+async function packageDemo(
+  platform: string,
+  architecture: string,
+  noCodeSigning = false
+) {
   doBuild(`${platform}-${architecture}`);
+
+  // third_party_licenses.txt
+  const licenseFilePath = path.join(outDir, thirdPartyNotices);
+  await new Promise((resolve, reject) => {
+    const licenseFile = fs.createWriteStream(licenseFilePath);
+    licenseFile.on("open", () => {
+      exec("yarn licenses generate-disclaimer --prod", (error, stdio) => {
+        if (error) {
+          return reject(error);
+        }
+        licenseFile.write(stdio);
+        licenseFile.close();
+        resolve(licenseFile);
+      });
+    });
+    licenseFile.on("error", (err) => {
+      reject(err);
+    });
+  });
+
   const extraOptions: Partial<packager.Options> = {};
-  if (platform === "darwin") {
-    const appleId = process.env.NOTARIZER_APPLE_ID;
-    const appleIdPassword = process.env.NOTARIZER_APPLE_ID_PASSWORD;
-    const signerIdentity = process.env.SIGNER_IDENTITY;
-    const shouldSignAndNotarize = false;
-    if (
-      appleId === undefined ||
-      appleIdPassword === undefined ||
-      signerIdentity == undefined
-    ) {
-      console.log(
-        "Specify NOTARIZER_APPLE_ID, NOTARIZER_APPLE_ID_PASSWORD, and SIGNER_IDENTITY in environment."
-      );
-      exit(1);
+  if (!noCodeSigning) {
```

Comment:
Since this is the only usage of the new parameter, I would invert the parameter name for readability to avoid having the "not" noCodeSigning double negative when reading.

Reply:
👍